cli/util/uploader.go

```diff
@@ -581,11 +582,19 @@ func uploadObj(ctx context.Context, p *pool.Pool, signer user.Signer, owner util
 	if err != nil {
 		return fmt.Errorf("failed to initiate object upload: %w", err)
 	}
-	defer writer.Close()
+
 	_, err = writer.Write(objData)
 	if err != nil {
 		return fmt.Errorf("failed to write object data: %w", err)
 	}
+	err = writer.Close()
+	if err != nil {
+		return fmt.Errorf("failed to close object writer: %w", err)
+	}
+	res := writer.GetResult()
+	if res.StoredObjectID().Equals(oid.ID(emptyOid)) {
```

Comment:
Why not just `oid.ID{}` instead of `oid.ID(emptyOid)`?

Reply:
Can be `oid.ID{}`, but `if res.StoredObjectID().Equals(oid.ID{})` check could be redundant.

Reply:
> check could be redundant.

Why? Is it guaranteed by NeoFS SDK that this OID is always non-empty?

Reply:
Not sure. But we even can calculate and set OID on our side before putting it.

Reply:
> Not sure. 

Can you check it?